synology_dsm/api/surveillance_station/camera.py

```diff
@@ -0,0 +1,99 @@
+"""SurveillanceStation camera."""
+from .const import RECORDING_STATUS, MOTION_DETECTION_DISABLED
+
+
+class SynoCamera(object):
+    """An representation of a Synology SurveillanceStation camera."""
+
+    def __init__(self, data, live_view_data=None):
+        """Initialize a Surveillance Station camera."""
+        self._data = data
+        self.live_view = SynoCameraLiveView(live_view_data)
+        self._motion_detection_enabled = None
+
+    def update(self, data):
+        """Update the camera."""
+        self._data = data
```

Comment:
`data` is coming from Camera_API.List

From your payload here: https://github.com/shenxn/python-synology/tree/surveillance_station/add/tests/api_data/dsm_6/surveillance_station

I don't see the `name`, `resolution`, `fps`, `recStatus` :thinking: 

What's wrong do you think @shenxn ?

Reply:
@Quentame I did some test and seems that those information are available in API version 1-7 and are removed in API version 8. It is wired since the documentation says those fields are available in API 9 and onward. But note that even those fields are in the table, they do not exist in the example output.

Reply:
`py-synology` uses version 1. This should be an easy fix.

Reply:
Actually the example in the doc does not provide those fields, so it's normal I think.

Multiple solutions:
- play with `basic` or `streamInfo` params of the `List` method
- uses `GetInfo` method so we get everything we want (this time the example is showing those fields)

I like the `GetInfo` way, but it will make one request more per camera, the thing that I would like to avoid at first place, perhaps it seems that we don't have the choice.

Also https://github.com/snjoetw/py-synology has a `camera_info` function with the `GetInfo` method, but I did not see the usage of it.

We need at least `name` (there is `newName` actually) and `recStatus`.
`resolution` and `fps` are my additions, but nice to have.

Reply:
Will use a max or fixed version if `GetInfo` leads to nothing.

Reply:
We can pass a list of ammera id to `GetInfo`, one one request then